### PR TITLE
988 funding updates

### PR DESCRIPTION
**Pull request by ZeusTheTrueGod**

### CODE REVIEW

File: tools/fundingHistoryData.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      return;
+    }
+    const previousEntry = _.find(prevItems, (prevItem) => prevItem.crunchbase_data && prevItem.crunchbase_data.name === item.crunchbase_data.name);
+    if (previousEntry && item.crunchbase_data.funding !== previousEntry.crunchbase_data.funding) {
+      result.push({
+        name: item.crunchbase_data.name,
+        currentAmount: item.crunchbase_data.funding,
+        previousAmount: previousEntry.crunchbase_data.funding,
+        date: date,
+        link: `https://l.cncf.io/organization=${saneName(item.crunchbase_data.name)}`,
```
Comment (dankohn) [inline tools/fundingHistoryData.js]: `landscape.cncf.io` (so we don't need a redirect)

File: tools/fundingHistoryPage.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  </style>
+</head>
+<body>
+     <table>
+     <thead>
+       <tr>
+         <td>Organization</td>
+         <td>Current Funding</td>
+         <td>Funding Change</td>
+         <td>Changing date</td>
```
Comment (dankohn) [inline tools/fundingHistoryPage.js]: Date of Change

File: tools/fundingHistoryPage.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+         <td>Crunchbase url</td>
```
Comment (dankohn) [inline tools/fundingHistoryPage.js]: Crunchbase URL

File: tools/fundingHistoryPage.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+</body>
+`;
+require('fs').writeFileSync('dist/funding.html', page);
+
+
+import { Feed } from "feed";
+const feed = new Feed({
+  title: "Funding changes",
+  description: "That feed shows recent changes",
+  id: "https://l.cncf.io/",
```
Comment (dankohn) [inline tools/fundingHistoryPage.js]: `landscape.cncf.io` everywhere, please.

File: tools/fundingHistoryPage.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  author: {
+    name: "CNCF",
+    email: "info@cncf.io",
+    link: "https://cncf.io"
+  }
+});
+
+result.forEach(item => {
+  const delta = item.currentAmount - (item.previousAmount || 0);
+  const text = `${item.name} changed funding at ${item.date} to $${millify(item.currentAmount)} by $${millify(delta)}` ;
```
Comment (dankohn) [inline tools/fundingHistoryPage.js]:
```
const text = `${item.name} got funding of $${millify(delta)}, now totalling $${millify(item.currentAmount)}, on ${item.date} `
```

File: tools/fundingHistoryPage.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+         <td>Funding Change</td>
+         <td>Changing date</td>
+         <td>Crunchbase url</td>
+       </tr>
+     </thead>
+         ${result.map(function(item, index) {
+           const delta = item.currentAmount - (item.previousAmount || 0);
+           return `
+               <tr style="${index % 2 === 0 ? '' : 'background: #eee'}">
+                 <td><a href="${item.link}" target="_blank">${item.name}</a></td>
```
Comment (dankohn) [inline tools/fundingHistoryPage.js]: Links should be of the form: https://landscape.cncf.io/grouping=organization&organization=hashi-corp

**Comment by ZeusTheTrueGod** [comment]
@dankohn , LGTM

**Pull request merged**